README.md

```diff
@@ -214,12 +214,14 @@ require("telescope").extensions.rest.select_env()
 - env_pattern: For env file pattern
 - env_edit_command: For env file edit command
 
+If running Ubuntu or Debian based systems you might need to run `ln -s $(which fdfind) ~/.local/bin/fd` to get extension to work. This becuase extension runs the [fd](https://github.com/sharkdp/fd?tab=readme-ov-file#installation) command.
```

Comment:
There's a small typo in `This becuase`, I think you meant `because` :p

Reply:
fixed